Dart/resources/META-INF/plugin.xml

```diff
@@ -159,6 +159,7 @@
     <annotator language="Dart" implementationClass="com.jetbrains.lang.dart.ide.annotator.DartColorAnnotator"/>
     <externalAnnotator language="Dart" implementationClass="com.jetbrains.lang.dart.analyzer.DartServerErrorsAnnotator"/>
     <externalAnnotator language="HTML" implementationClass="com.jetbrains.lang.dart.analyzer.DartServerErrorsAnnotator"/>
+    <externalAnnotator language="yaml" implementationClass="com.jetbrains.lang.dart.analyzer.DartServerErrorsAnnotator"/>
```

Comment:
I moved this to dart-yaml.xml. YAML plugin might be not installed or or disabled, so it's better to keep yaml-dependent extensions in a corresponding xml that is parsed only if YAML plugin in enabled.